sentry/resource_sentry_key.go

```diff
@@ -50,12 +50,16 @@ func resourceSentryKey() *schema.Resource {
 				Computed: true,
 			},
 			"rate_limit_window": {
-				Type:     schema.TypeInt,
-				Computed: true,
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: "time window in second for rate_limit_count",
+				Optional:    true,
 			},
 			"rate_limit_count": {
-				Type:     schema.TypeInt,
-				Computed: true,
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: "rate limit for the key in rate_limit_window time",
+				Optional:    true,
```

Comment:
What is the intend behind make this parameter optional? What these parameters have been used for before this change?

Reply:
Hi! Sorry for not mentioning the docs of the schema: https://www.terraform.io/docs/extend/schemas/index.html

The intent is that these parameters are not required to make the API working.
They are included in the GET request only and this PR will add it to the UPDATE request as well.